symbulate/distributions.py

```diff
@@ -1,10 +1,63 @@
 import numpy as np
+import scipy.stats as stats
+import matplotlib.pyplot as plt
 
 from .probability_space import ProbabilitySpace
+from .plot import configure_axes, get_next_color
+
+class Distribution(ProbabilitySpace):
+    def __init__(self, params, scipy, discrete = True):
+        self.params = params
+        
+        if discrete:
+            self.pdf = lambda x: scipy.pmf(x, **self.params)
+        else:
+            self.pdf = lambda x: scipy.pdf(x, **self.params)
+        
+        self.cdf = lambda x: scipy.cdf(x, **self.params)
+        self.quantile = lambda x: scipy.ppf(x, **self.params)
+        
+        self.median = lambda : scipy.median(**self.params)
+        self.mean = lambda : scipy.mean(**self.params)
+        self.var = lambda : scipy.var(**self.params)
+        self.sd = lambda : scipy.std(**self.params)
+        
+        self.discrete = discrete
+        
+        self.xlim = [
+            self.mean() - 3 * self.sd(), 
+            self.mean() + 3 * self.sd()
+            ]
+    
+    def plot(self, type = None, alpha = None, xlim = None, **kwargs):
+        if xlim is None: # if no limits for x-axis are specified, then use the default from plt
+            xlower, xupper = self.xlim
+        else:
+            xlower, xupper = xlim
+        
+        if self.discrete:
+            xlower = int(xlower)
+            xupper = int(xupper)        
+            xvals = list(np.arange(xlower, xupper+1, 1))
```

Comment:
Is there any reason you're casting this to a list instead of keeping it as a Numpy array?

Also, don't include the step when it's equal to 1. So just np.arange(xlower, xupper+1)